src/builds/builds.service.ts

```diff
@@ -99,18 +99,18 @@ export class BuildsService {
     return new BuildDto(build);
   }
 
-  async stop(id: string): Promise<BuildDto> {
+  async update(id: string, body: {}): Promise<BuildDto> {
```

Comment:
could we make separate type for it like `CreateBuildDto` ?

Reply:
This is a general update for one or multiple fields as JSON body.  If I were to create a separate type for the request, what would be the fields in that type as I would not know in advance what the consumer wants to update ? Also, this function returns the whole build information which is the same as BuildDto, so the return type looks good to me. Please let me know what you think.

Reply:
> what would be the fields in that type 

so far it would be only `ciBuildId` and `isRunning` both as optional params 

> Also, this function returns the whole build information which is the same as BuildDto, so the return type looks good to me

sure, everything is great with return type

Reply:
Thanks for the suggestion. I updated.